chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -0,0 +1,564 @@
+use serde::{Deserialize, Serialize};
+use sc_chain_spec::{ChainSpec, ChainType, ChainSpecExtension, ChainSpecGroup};
+use cumulus_primitives_core::ParaId;
+use sp_core::sr25519;
+use parachains_common::{AccountId, AssetHubPolkadotAuraId, AuraId, Balance};
+use crate::common::{testnet_accounts, get_from_seed, get_account_id_from_seed};
+
+/// Generic extensions for Parachain ChainSpecs.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ChainSpecGroup, ChainSpecExtension)]
+#[serde(deny_unknown_fields)]
+pub struct Extensions {
+	/// The relay chain of the Parachain.
+	pub relay_chain: String,
+	/// The id of the Parachain.
+	pub para_id: u32,
+}
+
+pub type AssetHubPolkadotChainSpec =
+	sc_service::GenericChainSpec<asset_hub_polkadot_runtime::RuntimeGenesisConfig, Extensions>;
```

Comment:
```suggestion
pub type AssetHubPolkadotChainSpec =
	ChainSpec<asset_hub_polkadot_runtime::RuntimeGenesisConfig, Extensions>;
```

Same for this and the others and then remove `sc-service`.